CvGameCoreDLL_Expansion2/CvCity.cpp

```diff
@@ -17551,7 +17551,7 @@ int CvCity::foodConsumptionNonSpecialistTimes100() const
 
 	int iFoodPerPop = /*2*/ GD_INT_GET(FOOD_CONSUMPTION_PER_POPULATION) * 100;
 	iFoodPerPop += GetAdditionalFood() * 100;
-	iFoodPerPop += GET_PLAYER(getOwner()).GetPlayerTraits()->GetNonSpecialistFoodChange();
+	iFoodPerPop += GET_PLAYER(getOwner()).GetNonSpecialistFoodChange() * 100 + GET_PLAYER(getOwner()).GetPlayerTraits()->GetNonSpecialistFoodChange() * 100;
```

Comment:
Looks like you added a multiplier to the Player Trait as well; the trait no longer needs to be times 100? Do we know any modmods that use this trait?

Reply:
You're right, it's used in a modmod with a decimal value. I reverted the change, and also adapted the minimum threshold